templates/snmpd.systemd_override.erb

```diff
@@ -0,0 +1,10 @@
+[Service]
```

Comment:
can you please provide new templates as an `erp()` template? From our [review docs](https://voxpupuli.org/docs/#reviewing-a-module-pr):
> Is a new template added? The preferred language is[ epp](https://puppet.com/docs/puppet/6.0/lang_template_epp.html), not erb

Reply:
Please also add a header, like `#THIS FILE IS MANAGED BY PUPPET`